pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2553,6 +2561,35 @@ func handlePhysicalIOAdapterListImpl(ctxArg interface{}, key string,
 			// We assume AddOrUpdateIoBundle will preserve any
 			// existing IsPort/IsPCIBack/UsedByUUID
 			aa.AddOrUpdateIoBundle(log, *ib)
+
+			// Adding PF before VF to have correct boot order
+			if ib.Type == types.PhyIoNetEthPF && ib.Vfs.Count > 0 {
+				ifNameMap, err := nettb.GetPciToIfNameMapByTimeout(60 * time.Second)
+				if err != nil {
+					log.Fatal("Failed to obtain ifNameMap")
+				}
+
+				err = createVF(ifNameMap[ib.PciLong], ib.Vfs.Count)
```

Comment:
Don't you need a check whether ib.PciLong exists in the map? Otherwise golang will panic.

Reply:
Removed, since we're using PciLongToIfName() from types/ifnametopci.go